fission-web-server/library/Fission/Web/Server/Handler/User/Verify.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
I removed every instance of this pragma from the code on my copy (there were 3 of them) and everything compiled. Can you remove this one and the others? Thanks!

Reply:
I know that GHC suggests this in some circumstances, especially while code is in flux. It's not the worst extension in the world, but also we shouldn't have it if we're not using it.

Reply:
:facepalm:  Yeah for some reason while code is in flux, I constantly need to add this extension to get the *next* compiler errors. But I frequently forget to remove it again. Will do!